V2_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,183 @@
+# V2 Migration Guide
+
+## Configuration
+
+### Required Keys
+
+- `appSession.secret` is now just `secret` (because it is now used to sign the transient cookies as well as the `appSession` cookie). The environmental variable has changed from `APP_SESSION_SECRET` to `SECRET`.
```

Comment:
```suggestion
- `appSession.secret` is now just `secret` (because it is now used to sign the transient cookies as well as the `appSession` cookie). The environment variable has changed from `APP_SESSION_SECRET` to `SECRET`.
```